{
    "url": "https://api.github.com/repos/pallets/flask/issues/1470",
    "repository_url": "https://api.github.com/repos/pallets/flask",
    "labels_url": "https://api.github.com/repos/pallets/flask/issues/1470/labels{/name}",
    "comments_url": "https://api.github.com/repos/pallets/flask/issues/1470/comments",
    "events_url": "https://api.github.com/repos/pallets/flask/issues/1470/events",
    "html_url": "https://github.com/pallets/flask/issues/1470",
    "id": 80355465,
    "node_id": "MDU6SXNzdWU4MDM1NTQ2NQ==",
    "number": 1470,
    "title": "route function quit unexpectedly",
    "user": {
        "login": "shuokay",
        "id": 5029591,
        "node_id": "MDQ6VXNlcjUwMjk1OTE=",
        "avatar_url": "https://avatars3.githubusercontent.com/u/5029591?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/shuokay",
        "html_url": "https://github.com/shuokay",
        "followers_url": "https://api.github.com/users/shuokay/followers",
        "following_url": "https://api.github.com/users/shuokay/following{/other_user}",
        "gists_url": "https://api.github.com/users/shuokay/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/shuokay/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/shuokay/subscriptions",
        "organizations_url": "https://api.github.com/users/shuokay/orgs",
        "repos_url": "https://api.github.com/users/shuokay/repos",
        "events_url": "https://api.github.com/users/shuokay/events{/privacy}",
        "received_events_url": "https://api.github.com/users/shuokay/received_events",
        "type": "User",
        "site_admin": false
    },
    "labels": [],
    "state": "closed",
    "locked": false,
    "assignee": null,
    "assignees": [],
    "milestone": null,
    "comments": 2,
    "created_at": "2015-05-25T02:47:18Z",
    "updated_at": "2015-05-25T15:10:30Z",
    "closed_at": "2015-05-25T15:10:30Z",
    "author_association": "NONE",
    "body": "When I call a python wrapper of C++ code in a flask route function, the flask route function quit unexpectedly. But if I call the wrapper from a normal python script ,everything goes well. Is there any condition that flask route function quit automatically?\n",
    "comments_inline": [
        {
            "url": "https://api.github.com/repos/pallets/flask/issues/comments/105160133",
            "html_url": "https://github.com/pallets/flask/issues/1470#issuecomment-105160133",
            "issue_url": "https://api.github.com/repos/pallets/flask/issues/1470",
            "id": 105160133,
            "node_id": "MDEyOklzc3VlQ29tbWVudDEwNTE2MDEzMw==",
            "user": {
                "login": "ThiefMaster",
                "id": 179599,
                "node_id": "MDQ6VXNlcjE3OTU5OQ==",
                "avatar_url": "https://avatars1.githubusercontent.com/u/179599?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/ThiefMaster",
                "html_url": "https://github.com/ThiefMaster",
                "followers_url": "https://api.github.com/users/ThiefMaster/followers",
                "following_url": "https://api.github.com/users/ThiefMaster/following{/other_user}",
                "gists_url": "https://api.github.com/users/ThiefMaster/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/ThiefMaster/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/ThiefMaster/subscriptions",
                "organizations_url": "https://api.github.com/users/ThiefMaster/orgs",
                "repos_url": "https://api.github.com/users/ThiefMaster/repos",
                "events_url": "https://api.github.com/users/ThiefMaster/events{/privacy}",
                "received_events_url": "https://api.github.com/users/ThiefMaster/received_events",
                "type": "User",
                "site_admin": false
            },
            "created_at": "2015-05-25T08:03:55Z",
            "updated_at": "2015-05-25T08:04:27Z",
            "author_association": "MEMBER",
            "body": "Define \"quit unexpectedly\".\n\nBut it sounds a lot like your native function is broken. Unlike Python code, bugs in C or C++ code can easily result in undefined behavior and sometimes crash while working in a slighly different environment.\n\nAlso, it's impossible for anyone to really help you with this unless you include the relevant code (preferably as a minimal snippet that reproduces the problem).\n"
        },
        {
            "url": "https://api.github.com/repos/pallets/flask/issues/comments/105249442",
            "html_url": "https://github.com/pallets/flask/issues/1470#issuecomment-105249442",
            "issue_url": "https://api.github.com/repos/pallets/flask/issues/1470",
            "id": 105249442,
            "node_id": "MDEyOklzc3VlQ29tbWVudDEwNTI0OTQ0Mg==",
            "user": {
                "login": "untitaker",
                "id": 837573,
                "node_id": "MDQ6VXNlcjgzNzU3Mw==",
                "avatar_url": "https://avatars0.githubusercontent.com/u/837573?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/untitaker",
                "html_url": "https://github.com/untitaker",
                "followers_url": "https://api.github.com/users/untitaker/followers",
                "following_url": "https://api.github.com/users/untitaker/following{/other_user}",
                "gists_url": "https://api.github.com/users/untitaker/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/untitaker/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/untitaker/subscriptions",
                "organizations_url": "https://api.github.com/users/untitaker/orgs",
                "repos_url": "https://api.github.com/users/untitaker/repos",
                "events_url": "https://api.github.com/users/untitaker/events{/privacy}",
                "received_events_url": "https://api.github.com/users/untitaker/received_events",
                "type": "User",
                "site_admin": false
            },
            "created_at": "2015-05-25T15:10:30Z",
            "updated_at": "2015-05-25T15:10:30Z",
            "author_association": "MEMBER",
            "body": "What @ThiefMaster said, and please provide a good testcase before saying this is a problem with Flask.\n"
        }
    ]
}